Source/WebInspectorUI/UserInterface/Views/CookieStorageContentView.js

```diff
@@ -455,7 +465,8 @@ WI.CookieStorageContentView = class CookieStorageContentView extends WI.ContentV
         let promises = [];
         if (editingCookie)
             promises.push(target.PageAgent.deleteCookie(editingCookie.name, editingCookie.url));
-        promises.push(target.PageAgent.setCookie(cookieProtocolPayload));
+        let needsPartitionKey = !cookieToSet.partitionKey && !!cookieToSet.partitioned ? true : undefined;
+        promises.push(target.PageAgent.setCookie(cookieProtocolPayload, needsPartitionKey));
```

Comment:
this will fail when inspecting older versions of iOS because `shouldPartition` doesn't exist
```js
// COMPATIBILITY (iOS 18.X, macOS 15.X): `Page.setCookie` did not have a `shouldPartition` parameter yet.
promises.push(target.PageAgent.setCookie.invoke({
    cookie: cookieProtocolPayload,
    shouldPartition: !cookieToSet.partitionKey && !!cookieToSet.partitioned,
}));
```

Reply:
I'll come back and fill in the `.X` later.

Reply:
FYI we normally do all of these at once when we snapshot a new protocol version